src/bluealsa-dbus.c

```diff
@@ -129,6 +133,7 @@ static void ba_variant_populate_pcm(GVariantBuilder *props, const struct ba_tran
 	g_variant_builder_add(props, "{sv}", "Delay", ba_variant_new_pcm_delay(pcm));
 	g_variant_builder_add(props, "{sv}", "SoftVolume", ba_variant_new_pcm_soft_volume(pcm));
 	g_variant_builder_add(props, "{sv}", "Volume", ba_variant_new_pcm_volume(pcm));
+	g_variant_builder_add(props, "{sv}", "Sequence", ba_variant_new_pcm_sequence(pcm));
```

Comment:
Put this new property after the `Device` - I think it's more related to device than transport. Please, make the same change in the API txt file and in the bluealsa-iface.c.

Reply:
done